guides/common/modules/ref_glossary-of-terms.adoc

```diff
@@ -0,0 +1,367 @@
+[id="glossary-of-terms_{context}"]
+= Glossary of terms
+
+This glossary documents various terms used in relation to {ProjectName}.
```

Comment:
```suggestion
[id="glossary-of-terms-used-in-project_{context}"]
= Glossary of terms used in {Project}
```
WDYT?

Reply:
Like it!

Reply:
9001dc44f7b58386e5bf0ffd33953fd15b9047f0